crates/oxc_minifier/src/peephole/replace_known_methods.rs

```diff
@@ -1289,25 +1316,27 @@ mod test {
     }
 
     #[test]
-    #[ignore]
     fn test_fold_math_functions_max() {
-        test_same("Math.max(Math.random(), 1)");
+        test_same_value("Math.max(Math.random(), 1)");
 
-        test("Math.max()", "-Infinity");
-        test("Math.max(0)", "0");
-        test("Math.max(0, 1)", "1");
-        test("Math.max(0, 1, -1, 200)", "200");
+        test_value("Math.max()", "-Infinity");
+        test_value("Math.max(0)", "0");
+        test_value("Math.max(0, 1)", "1");
+        test_value("Math.max(0, 1, -1, 200)", "200");
+        test_value("Math.max(0, -1, -Infinity)", "0");
+        test_value("Math.max(0, -1, -Infinity, NaN)", "NaN");
     }
 
     #[test]
-    #[ignore]
     fn test_fold_math_functions_min() {
-        test_same("Math.min(Math.random(), 1)");
-
-        test("Math.min()", "Infinity");
-        test("Math.min(3)", "3");
-        test("Math.min(0, 1)", "0");
-        test("Math.min(0, 1, -1, 200)", "-1");
+        test_same_value("Math.min(Math.random(), 1)");
+
+        test_value("Math.min()", "Infinity");
+        test_value("Math.min(3)", "3");
```

Comment:
```suggestion
        test_value("Math.min(3)", "3");
        test_value("Math.min(0, -0)", "-0");
        test_value("Math.min(-0, 0)", "-0");
```

Reply:
It seems this test is failing. Maybe we need a special handling when the argument includes `-0`.